rustfmt-core/rustfmt-bin/src/bin/main.rs

```diff
@@ -704,7 +650,7 @@ mod test {
         let output = child
             .wait_with_output()
             .expect("Failed to wait on rustfmt child");
-        assert!(output.status.success());
+        assert!(!output.status.success());
```

Comment:
Nice! Did not realize we had a subtle bug here before that is now fixed with this PR